src/pages/Home.page.vue

```diff
@@ -66,10 +75,20 @@ const { t } = useI18n();
         <div v-if="toolStore.favoriteTools.length > 0">
           <h3 class="mb-5px mt-25px font-500 text-neutral-400">
             {{ $t('home.categories.favoriteTools') }}
+            <c-tooltip tooltip="Drag and drop to reorder favorites">
+              <n-icon :component="DragDrop" size="18" />
+            </c-tooltip>
           </h3>
-          <div class="grid grid-cols-1 gap-12px lg:grid-cols-3 md:grid-cols-3 sm:grid-cols-2 xl:grid-cols-4">
-            <ToolCard v-for="tool in toolStore.favoriteTools" :key="tool.name" :tool="tool" />
-          </div>
+          <Draggable
+            :list="favoriteTools"
+            class="grid grid-cols-1 gap-12px lg:grid-cols-3 md:grid-cols-3 sm:grid-cols-2 xl:grid-cols-4"
+            item-key="name"
+            @end="onUpdateFavoriteTools"
+          >
```

Comment:
Maybe also add a gost class for better ux

```suggestion
          <Draggable
            :list="favoriteTools"
            class="grid grid-cols-1 gap-12px lg:grid-cols-3 md:grid-cols-3 sm:grid-cols-2 xl:grid-cols-4"
            item-key="name"
            @end="onUpdateFavoriteTools"
            ghost-class="..."
          >
```

Reply:
nice idea - updated to include the ghost-class:
![demo](https://github.com/user-attachments/assets/58b99142-c2eb-4152-a5e2-4e2812c1263c)